components/Navbar.js

```diff
@@ -206,10 +217,7 @@ const StyledDiv = styled.div`
   width: 35px;
   height: 35px;
   background-color: ${props =>
-    props.active === true ? "rgba(255, 123, 137)" : ""};
-`;
-
-const StyledButton = styled.button`
-  background-color: transparent;
-  border: none;
+    props.active === true
+      ? /* "rgba(165, 202, 210, 0.5)" */ "rgba(255, 123, 137, 0.3)"
```

Comment:
Please delete code you don't use anymore.

Reply:
done!